person.rb

```diff
@@ -0,0 +1,47 @@
+class Person
+  # Initialize
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    @age = age
+    @name = name
+    @parent_permission = parent_permission
+    @id = rand(0..1000)
+  end
+
+  # Getters
+  def person_age
+    @age
+  end
+
+  def person_id
+    @id
+  end
+
+  def person_name
+    @name
+  end
+
+  # Setters
+  def def_age(age)
+    @age = age
+  end
+
+  def def_name(name)
+    @name = name
+  end
```

Comment:
**OPTIONAL** - Good job implementing, the getters and setters for your class, I suggest an easier way would be to use [shortcuts](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_principles_examples.md#useful-shortcuts-attr_reader-attr_writer-and-attr_accessor). Please fix this 🙏